src/librustc_error_codes/error_codes/E0271.md

```diff
@@ -70,13 +63,17 @@ fn foo<T>(t: T) where T: Trait<AssociatedType = &'static str> {
 impl Trait for i8 { type AssociatedType = &'static str; }
 
 foo(3_i8);
+```
+
+2) in the `Trait` implementation for `i8`
+```
+trait Trait { type AssociatedType; }
 
-// For-Loop Example:
-let vs = vec![1, 2, 3, 4];
-for v in &vs {
-    match v {
-        &1 => {}
-        _ => {}
-    }
+fn foo<T>(_t: T) where T: Trait<AssociatedType = u32> {
```

Comment:
```suggestion
fn foo<T>(t: T) where T: Trait<AssociatedType = u32> {
```

Reply:
The parameter is never used, so it should be prefixed with an underscore. See https://play.rust-lang.org/?version=stable&mode=debug&edition=2018&gist=25a2cafb2b889bac1dbfe58fea32bedb

Reply:
We don't mind if it's unused or not here. Actually, we don't add underscores to the above snippets and other error codes' snippets.